coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/provider/util/JsonbUtil.java

```diff
@@ -62,18 +62,24 @@ public class JsonbUtil {
      */
     public static final String JSONB_CONFIG_PREFIX = "coffee.jsonb.config";
 
-    /**
-     * Create {@link Jsonb} instance with {@link FieldOnlyVisibilityStrategy} property visibility strategy.
-     * 
-     * @return configured {@link Jsonb} instance
-     */
-    public static Jsonb getContext() {
+    private static final Jsonb JSONB;
+
+    static {
         Config config = ConfigUtil.getInstance().defaultConfig();
         JsonbConfig jsonbConfig = new JsonbConfig()
                 // property visibility strategy setting
                 .withPropertyVisibilityStrategy(getPropertyVisibilityStrategyClass(config))
                 .withBinaryDataStrategy(getBinaryDataStrategy(config));
-        return JsonbBuilder.newBuilder().withConfig(jsonbConfig).build();
+        JSONB = JsonbBuilder.newBuilder().withConfig(jsonbConfig).build();
```

Comment:
release notes + there should be a solution to override, add new elements to the config

Reply:
overriding and new custom element configuration was implemented in pull request https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/pull/702/files#diff-b36c3e6207f73415def2a553a995e41a13735c7c0d71636022f4f8fc1b3bae70R81